ui/job-view/details/DetailsPanel.jsx

```diff
@@ -20,6 +23,12 @@ import PinBoard from './PinBoard';
 import SummaryPanel from './summary/SummaryPanel';
 import TabsPanel from './tabs/TabsPanel';
 
+Glean.setLogPings(true);
+if (!origin.includes('treeherder.mozilla.org')) {
```

Comment:
Does the frontend know about environment variables? If so, you could set something like `NODE_ENV='production'` and check that instead?

Reply:
I couldn't figure that out, so instead I will as mentioned in a previous comment, comment out the debugview stuff.